src/app.js

```diff
@@ -41,6 +42,11 @@ export function view(state, actions) {
         render: ({location}) =>
           Question({id: parseQuestionIdFromPath(location.pathname), data: state.data}),
       }),
+      Route({
+        path: '/ask',
+        render: () =>
```

Comment:
you can make this a one liner, you only split it if the line is too long
`render: () => Ask(),`